json-body-transformer/src/test/java/com/ninecookies/wiremock/extensions/JsonBodyTransformerTest.java

```diff
@@ -309,11 +333,21 @@ public void doesNotApplyGlobally() throws Exception {
 		wireMockServer.verify(postRequestedFor(urlEqualTo(REQUEST_URL)));
 	}
 
-	@Test
-	public void injectRandom() {
+	@DataProvider
+	private Object[][] randomFormats() {
+		return new Object[][] {
+				{ "$(!Random)" },
+				{ "$(!Random.1)" },
+				{ "$(!Random.abc)" },
+				{ "$(!Random.ABC)" },
+		};
+	}
+
+	@Test(dataProvider = "randomFormats")
+	public void injectRandom(String format) {
 		wireMockServer.stubFor(post(urlEqualTo(REQUEST_URL))
 				.willReturn(aResponse().withStatus(200).withHeader("content-type", CONTENT_TYPE)
-						.withBody("{\"randomNumber\":\"$(!Random)\", \"got\":\"it\"}")
+						.withBody("{\"randomNumber\":\"" + format + "\", \"got\":\"it\"}")
```

Comment:
I don't get that. Where are the `.1`, `.abc` and `.ABC` checked? The following checks are just checking whether there is an integer returned. Shouldn't that result in something like `{ "randomNumber": "43432.abc" }`?